quarkus/extensions/kogito-quarkus-workflow-extension-common/kogito-quarkus-workflow-common/src/main/java/org/kie/kogito/quarkus/workflow/conf/KogitoPersistenceBuildTimeConfig.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.kie.kogito.quarkus.conf;
+package org.kie.kogito.quarkus.workflow.conf;
```

Comment:
```suggestion
package org.kie.kogito.quarkus.workflow.config;
```

don't we have a similar package name somewhere here?

Reply:
done, yeah, I created another pr to handle it and get to some uniformity https://github.com/kiegroup/kogito-runtimes/pull/2683